lib/web/underscore.js

```diff
@@ -1327,7 +1327,8 @@
     '>': '&gt;',
     '"': '&quot;',
     "'": '&#x27;',
-    '`': '&#x60;'
+    '`': '&#x60;',
+    "'": '&#039;',
```

Comment:
Hi @vinayshah. Underscore.js is a separate library, we can't make changes to it. I think we can add a wrapper to this method inside `mageUtils`.